content/Unreal Tournament 2004/Models/M/a/3/93c3e6/midgetpsychobandit_[a393c3e6].yml

```diff
@@ -0,0 +1,51 @@
+--- !<MODEL>
+contentType: "MODEL"
+firstIndex: "2022-05-28 16:06"
+game: "Unreal Tournament 2004"
+name: "MidgetPsychoBandit"
+author: "Unknown"
+description: "None"
+releaseDate: "2014-03"
+attachments:
+- type: "IMAGE"
+  name: "midgetpsychobandit_shot_a393c3e6_2.png"
+  url: "https://f002.backblazeb2.com/file/unreal-archive-images/Unreal%20Tournament%202004/Models/M/a/3/93c3e6/midgetpsychobandit_shot_a393c3e6_2.png"
+- type: "IMAGE"
+  name: "midgetpsychobandit_shot_a393c3e6_1.png"
+  url: "https://f002.backblazeb2.com/file/unreal-archive-images/Unreal%20Tournament%202004/Models/M/a/3/93c3e6/midgetpsychobandit_shot_a393c3e6_1.png"
+- type: "IMAGE"
+  name: "midgetpsychobandit_shot_a393c3e6_3.png"
+  url: "https://f002.backblazeb2.com/file/unreal-archive-images/Unreal%20Tournament%202004/Models/M/a/3/93c3e6/midgetpsychobandit_shot_a393c3e6_3.png"
+originalFilename: "MidgetBanditBeta1.zip"
+hash: "a393c3e6580c4b614afa175f8466e5d433cf8934"
+fileSize: 6947712
+files:
+- name: "MidgetPGtext.utx"
+  fileSize: 6558429
+  hash: "85316ecd001fb266e89efbac4e7d4bc1f476ddff"
+- name: "MidgetBanditPGBeta1.upl"
+  fileSize: 1777
+  hash: "2af4745f271e546cd29ceed05c2428dc17509dd8"
+- name: "MidgetBanditAnim.ukx"
+  fileSize: 4259366
+  hash: "2d34c510f738f21981ea41d5df53486f55d9eb48"
+otherFiles: 1
+dependencies:
+  MidgetBanditAnim.ukx:
+  - status: "OK"
+    name: "MidgetPGtext"
+downloads:
+- url: "https://f002.backblazeb2.com/file/unreal-archive-files/Unreal%20Tournament%202004/Models/M/a/3/93c3e6/MidgetBanditBeta1.zip"
+  main: true
+  repack: false
+  state: "OK"
+links: {}
+deleted: false
+models:
+- "MidgetPsychoBandit"
```

Comment:
Model and Skin names:
Midget Psycho Bandit (BL2)
Midget Loot Psycho (BL2)
Midget Suicidal Psycho (BL2)